### PR TITLE
test: enable typescript tests

### PR DESCRIPTION
**Pull request by mattmazzola**
Moved the typescript tests to a folder so they can be more easily run via cypress --spec path.

Need to start running these regularly to start to get sense of what is commonly broken in these tests to make them more reliable.  I noticed the smoke test broke after Lars recent PR and not sure why or what else broke.

I wanted to have this in a separate circleCI job so that this could be a whole separate condition and not make the cypress:test misleading (failing due to ts tests when normal regression tests pass) but there doesn't seem to be a way to re-use existing job state and the cost of recloning repos / installing packages / starting servers etc is too expensive to repeat just to get a separate check mark.

### CODE REVIEW

**Comment by mattmazzola** [comment]
Was looking more in config optimizations.

Seems we could potentially use https://circleci.com/docs/2.0/configuration-reference/#persist_to_workspace to eliminate the repeat of cloning, installing, building, etc steps however we would still need to start both servers and there isn't much benefit other than having a separate / independent checkmark

File: cypress/integration/regressionTs/setEntityActions.spec.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+import * as util from '../../support/utilities'
+import constants from '../../support/constants'
+import s from '../../support/selectors'
 
 describe('Set Entity Actions', () => {
     const testData = {
         modelName: `setEntity`,
         modelFile: 'setEntity.cl',
-        enumValueCount: 14,
+        enumValueCount: 11, // 14,
```
Comment (mattmazzola) [inline cypress/integration/regressionTs/setEntityActions.spec.ts]: I'm not sure why 14 worked previously and fails now, but 11 seems correct. Maybe we fixed a bug and this was dependent on that bug.

File: cypress/support/selectors.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ const selectors = {
         buttonLocalFile: '[data-testid="model-creator-locate-file-button"]',
         name: '[data-testid=model-creator-input-name]',
         submit: '[data-testid=model-creator-submit-button]',
-        inputFile: '[data-testid="model-creator-import-file-picker"] > div > input[type="file"]',
+        inputFile: '[data-testid="model-creator-import-file-picker"] input[type="file"]',
```
Comment (mattmazzola) [inline cypress/support/selectors.ts]: Similar 6.x change as previous PR https://github.com/microsoft/ConversationLearner-UI/pull/1201

**Comment by mattmazzola** [comment]
Do we still use any of the other top level tests such as `(All)` or `(Regression)` or `(RunTestsFromLists)`? It would seem any temporary things already go in `zTemp`

**Comment by SDESkowronski** [comment]
No, I’m using other techniques to accomplish what these were meant to achieve. They can be removed…I’ll remove them in my current work.

**Pull request merged**